app/src/main/res/layout/pocket_video_mega_tile.xml

```diff
@@ -10,38 +10,90 @@
-        style="@style/PocketVideoMegaTileThumbnail"
-        android:contentDescription="@null"
-        />
-
-    <ImageView
-        android:id="@+id/thumbnail2View"
-        style="@style/PocketVideoMegaTileThumbnail"
-        android:contentDescription="@null"
-        />
-
-    <ImageView
-        android:id="@+id/thumbnail3View"
-        style="@style/PocketVideoMegaTileThumbnail"
-        android:contentDescription="@null"
-        />
-
-    <ImageView
-        android:id="@+id/thumbnail4View"
-        style="@style/PocketVideoMegaTileThumbnail"
-        android:contentDescription="@null"
-        />
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:id="@+id/pocketVideosContainer"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:layout_gravity="center">
+
+            <ImageView
+                android:id="@+id/pocketWordmarkView"
+                android:layout_width="0dp"
+                android:layout_height="30.5dp"
+                android:layout_weight="1"
+                android:layout_gravity="center_vertical"
+                android:contentDescription="@null"
+                tools:src="@drawable/ic_pocket_and_wordmark"
+                tools:tint="@color/photonGrey10"
+                />
+
+            <ImageView
+                android:id="@+id/thumbnail1View"
+                style="@style/PocketVideoMegaTileThumbnail"
+                android:contentDescription="@null"
+                />
+
+            <ImageView
+                android:id="@+id/thumbnail2View"
+                style="@style/PocketVideoMegaTileThumbnail"
+                android:contentDescription="@null"
+                />
+
+            <ImageView
+                android:id="@+id/thumbnail3View"
+                style="@style/PocketVideoMegaTileThumbnail"
+                android:contentDescription="@null"
+                />
+
+            <ImageView
+                android:id="@+id/thumbnail4View"
+                style="@style/PocketVideoMegaTileThumbnail"
+                android:contentDescription="@null"
+                />
+
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/pocketErrorContainer"
+            android:layout_height="wrap_content"
+            android:layout_width="wrap_content"
+            android:orientation="vertical"
+            android:layout_gravity="center"
+            android:gravity="center"
+            android:visibility="gone"
+            >
+
+            <TextView
+                android:id="@+id/pocketMegaTileLoadError"
+                android:layout_height="wrap_content"
+                android:layout_width="wrap_content"
+                android:text="@string/pocket_video_feed_failed_to_load"
+                android:layout_marginTop="16dp"
```

Comment:
It's generally better practice to refactor and keep layout margins in /res/values/dimens.xml